src/js/components/ToggleCheckboxList/ToggleCheckboxList.tsx

```diff
@@ -25,6 +25,7 @@ type Props = {
   headerStatus: object;
   setActiveRows: React.Dispatch<React.SetStateAction<{}>>;
   setHeaderStatus: React.Dispatch<React.SetStateAction<{}>>;
+  sortOrder?: string;
```

Comment:
thoughts on adding an enum to describe sort order? We could probably add that to the types folder in v4.